test/e2e/network_test.go

```diff
@@ -354,7 +357,14 @@ func (tc *testContext) createWindowsServerDeployment(name string, command []stri
 							Name:            name,
 							Image:           windowsServerImage,
 							ImagePullPolicy: v1.PullIfNotPresent,
-							Command:         command,
+							// The default user for nanoserver image is ContainerUser.
+							// Change user to ContainerAdministrator to run HttpListener in admin mode.
+							SecurityContext: &v1.SecurityContext{
```

Comment:
I think [this](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#windowssecuritycontextoptions-v1-core) feature is stable in 1.18 and we should check if we need to explicitly enable any dependet features at the featuregate level.

Reply:
[`WindowsRunAsUserName`](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) is enabled by default. We would not need to explicitly enable anything.  https://kubernetes.io/docs/tasks/configure-pod-container/configure-runasusername/#set-the-username-for-a-container